ui/raidboss/data/06-ew/raid/p2s.txt

```diff
@@ -0,0 +1,118 @@
+### P2S: Asphodelos: The Second Circle (Savage)
+#
+# -oc Hippokampos -ii 6822 6821 6829 6D4A
+
+hideall "--Reset--"
+hideall "--sync--"
+
+0.0 "--Reset--" sync / 21:........:40000010:/ window 100000 jump 0
+
+0.0 "--sync--" sync /Engage!/ window 0,1
+11.0 "--sync--" sync / 14:[^:]*:Hippokampos:6833:/ window 15,15
+16.0 "Murky Depths" sync / 1[56]:[^:]*:Hippokampos:6833:/
+26.3 "Doubled Impact" sync / 1[56]:[^:]*:Hippokampos:6832:/
+31.5 "--middle--" sync / 1[56]:[^:]*:Hippokampos:6836:/
+38.8 "Sewage Deluge" sync / 1[56]:[^:]*:Hippokampos:6810:/
+48.0 "--middle--" sync / 1[56]:[^:]*:Hippokampos:6836:/
+61.8 "Spoken Cataract/Winged Cataract" sync / 1[56]:[^:]*:Hippokampos:(6811|6812|6813|6814|6815|6816|6817|6818):/
+81.9 "Coherence" sync / 1[56]:[^:]*:Hippokampos:681B:/
+82.6 "Coherence Flare" sync / 1[56]:[^:]*:Hippokampos:681A:/
+85.1 "Coherence Line" sync / 1[56]:[^:]*:Hippokampos:681C:/
+87.2 "--sync--" sync / 1[56]:[^:]*:Hippokampos:681D:/
+98.4 "Murky Depths" sync / 1[56]:[^:]*:Hippokampos:6833:/
+105.1 "Ominous Bubbling" sync / 1[56]:[^:]*:Hippokampos:682A:/
+115.9 "Shockwave" sync / 1[56]:[^:]*:Hippokampos:682F:/
+119.7 "Ominous Bubbling Groups" sync / 1[56]:[^:]*:Hippokampos:682B:/
+
+# Predatory Avarice and Channeling Flow
+128.1 "--middle--" sync / 1[56]:[^:]*:Hippokampos:6836:/
+134.5 "Predatory Avarice" sync / 1[56]:[^:]*:Hippokampos:6827:/
+152.2 "Winged Cataract/Spoken Cataract" sync / 1[56]:[^:]*:Hippokampos:(6811|6812|6813|6814|6815|6816|6817|6818):/
+158.5 "Hard Water/Deadly Current" sync / 1[56]:[^:]*:Hippokampos:(6828|6829):/
+171.9 "Channeling Flow" sync / 1[56]:[^:]*:Hippokampos:681E:/
+187.2 "Great Typhoon" sync / 1[56]:[^:]*:Hippokampos:(6820|6822):/
+187.7 "Crash" #sync / 1[56]:[^:]*:Hippokampos:6821:/
+202.2 "Doubled Impact" sync / 1[56]:[^:]*:Hippokampos:6832:/
+212.3 "Murky Depths" sync / 1[56]:[^:]*:Hippokampos:6833:/
+
+# Kampeos Harma
+221.5 "--middle--" sync / 1[56]:[^:]*:Hippokampos:6836:/
+228.9 "Sewage Deluge" sync / 1[56]:[^:]*:Hippokampos:6810:/
+246.4 "Shockwave" sync / 1[56]:[^:]*:Hippokampos:682F:/
+259.3 "Kampeos Harma" sync / 1[56]:[^:]*:Hippokampos:6823:/
+259.7 "Kampeos Harma 1" #sync / 1[56]:[^:]*:Hippokampos:(6825|6824):/
+261.2 "Kampeos Harma 2 (head)" #sync / 1[56]:[^:]*:Hippokampos:6825:/

```

Comment:
FWIW, I think you could skip the (head) lines here, and only have the charges, since those are the only ones people have to move on and the boss is untargetable.

Reply:
I kept the 2826 as it is a syncable point, but changed it to be hidden.